Website/src/rendering/src/assets/css/layout/sections.css

```diff
@@ -59,68 +23,72 @@
     @apply bg-white;
 
     .section__content {
-      .section__content__p {
-        @apply text-black;
-      }
-
       .section__content__title {
-        @apply text-blue normal-case;
+        @apply normal-case;
```

Comment:
`normal-case` should be removed from the `--light` and `--dark` variants and instead added to the default styling of titles where `text-blue-light` is also defined.